lib/rubocop/cop/metrics/block_nesting.rb

```diff
@@ -43,9 +49,22 @@ def check_nesting_level(node, max, current_level)
           end
         end
 
+        def consider_node?(node)
+          return true if NESTING_BLOCKS.include?(node.type)
+
+          case style
+          when :multiline then node.block_type? && node.multiline?
+          when :all then node.block_type?
+          end
+        end
+
         def message(max)
           "Avoid more than #{max} levels of block nesting."
         end
+
+        def style_parameter_name
```

Comment:
Seems odd to be using the `style`-related code for something that's not the cop's enforced style, but an additional config option. @jonas054 Thoughts?